umbraco/Umbraco-CMS
# v9: don't reload page when navigating to edit user

Fixes and issue where when you pressed the edit user button:
![image](https://user-images.githubusercontent.com/70372949/139023129-79df3785-3a12-4486-b1f4-ab195fd770c0.png)

This would trigger a full page reload, and thus show tours/email subscription notifications again.

# Notes 
- Changed button to now use editUser function
- implemented editUser function

# How to test
- Run a clean install of umbraco
- When you get the tour notification, just close it (**do not press "don't show this again"**)
- Press the edit user button, it should not give you the tours popup anymore
